tests/daily_scraper/test_invalid_platform.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from mozdownload import DailyScraper
+import mozdownload.errors as errors
+
+@pytest.mark.parametrize('args', [
+    ({'application': 'fenix', 'platform': 'mac'}),
```

Comment:
Please note that with the follow-up PR for `NotSupportedError` this will change given that `mac` is a not supported platform. But I'm fine with landing this for now, given that it only affects Fenix builds.